src/commands/realPress.ts

```diff
@@ -14,58 +14,15 @@ export interface RealPressOptions {
   log?: boolean;
 }
 
-function getKeyDefinition(key: keyof typeof keyCodeDefinitions) {
-  const keyDefinition = keyCodeDefinitions[key];
-
-  if (!keyDefinition) {
-    throw new Error(`Unsupported key '${key}'.`);
-  }
-
-  const keyCode = keyDefinition.keyCode ?? 0;
-  return {
-    keyCode: keyCode,
-    key: keyDefinition?.key ?? "",
-    text: keyDefinition.key.length === 1 ? keyDefinition.key : undefined,
```

Comment:
this line is why this was needed: https://github.com/dmtrKovalenko/cypress-real-events/commit/5978a6e3efa92b63e8d1ac31eac907d092aaebd3 , after the change to:
```diff
-    text: keyDefinition.key.length === 1 ? keyDefinition.key : undefined,
+    text: 'text' in keyDefinition ? keyDefinition.text : undefined,
```
the special-casing of Enter is not needed anymore, so I've basically reverted the referenced commit